frontend/src/components/landingPage/SearchPart.vue

```diff
@@ -33,6 +33,11 @@
     <div>
       <v-app>
         <div id="resultTableDiv" class="card">
+          <div
+            class="inputQueryInResult"
+            v-if="readyToSearchQuery"
+          >you searched for the query : {{ readyToSearchQuery }}</div>

```

Comment:
Correct grammar here.... Also its only visible shortly while the results are loading... Looks a bit weird...

Reply:
This has been fixed and commit has been pushed.

Reply:
public/fonts has been ignored now.

Reply:
The files are still in the changeset though...

Reply:
The files have been removed now. Also, this won`t get generated in the future, as it has been added to gitIgnore